src/store/facade/GroceryStore.java

```diff
@@ -204,22 +205,6 @@ public Order searchById(String id) {
 			return null;
 		}
 
-		/**
-		 * Gets a list of outstanding (not yet fulfilled) orders.
-		 * 
-		 * @return an iterator to the list of outstanding orders
-		 */
-		public Iterator<Order> outstanding() {
```

Comment:
removed + outstanding(): Iterator<Order>

Reply:
Yeah, I think Ben found his own way to deal with the list of outstanding orders, so this one was redundant.